web-frontend/src/pages/Assets/Assets.tsx

```diff
@@ -1,24 +1,62 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 
-import { Button } from '@massalabs/react-ui-kit';
+import { Button, toast } from '@massalabs/react-ui-kit';
+import { AxiosError } from 'axios';
 import { FiPlus } from 'react-icons/fi';
 import { useParams } from 'react-router-dom';
 
 import { AssetsImportModal } from './AssetsImportModal';
 import { AssetsList } from './AssetsList';
 import { AssetsLoading } from './AssetsLoading';
-import { useResource } from '@/custom/api';
+import { assetImportErrors } from '@/const/assets/assets';
+import { usePost, useResource } from '@/custom/api';
 import Intl from '@/i18n/i18n';
 import { WalletLayout, MenuItem } from '@/layouts/WalletLayout/WalletLayout';
 import { IToken } from '@/models/AccountModel';
+import { ImportAssetsObject } from '@/models/assetModel';
 
 function Assets() {
   const [modal, setModal] = useState(false);
+
   const { nickname } = useParams();
   const { data: tokenArray = [], isLoading: isGetLoading } = useResource<
     IToken[]
   >(`accounts/${nickname}/assets`);
 
+  const {
+    mutate,
+    isSuccess: postSuccess,
+    isError: postError,
+    error,
+  } = usePost<ImportAssetsObject>(`accounts/${nickname}/assets`);
+
+  const axiosError = error as AxiosError;
+  const postErrorStatus = axiosError?.response?.status;
+
+  useEffect(() => {
+    if (postSuccess) {
+      toast.success(Intl.t('assets.success'));
+      setModal(false);
+    } else if (postError) {
+      switch (postErrorStatus) {
+        case assetImportErrors.badRequest:
+          toast.error(Intl.t('assets.bad-request'));
+          break;
+        case assetImportErrors.invalidAddress:
+          toast.error(Intl.t('assets.invalid-address'));
+          break;
+        case assetImportErrors.notFound:
+          toast.error(Intl.t('assets.not-found'));
+          break;
+        case assetImportErrors.serverError:
+          toast.error(Intl.t('assets.internal-server-error'));
+          break;
+        default:
+          console.log('Unknown Error:', postErrorStatus);
+      }
```

Comment:
Can you put these lines into a function name `getError` or `displayError`? (this function take as parameter `postErrorStatus`) 🤔